opmd_viewer/openpmd_timeseries/plotter.py

```diff
@@ -8,13 +8,19 @@
 Author: Remi Lehe
 License: 3-Clause-BSD-LBNL
 """
+import numpy as np
 try:
     import warnings
     import matplotlib
     import matplotlib.pyplot as plt
     matplotlib_installed = True
 except ImportError:
     matplotlib_installed = False
+try:
+    from .cython_function import histogram_cic_1d, histogram_cic_2d
```

Comment:
just FYI on the risks of `try import` of `Cython` without using the import during the try block: https://github.com/pypa/setuptools/issues/1229